File: FE/src/types/ReviewType.ts
export interface ReviewType {
    reviewId?: number;
    senderId: number;
    recipientId: number;
    contractId: number;
    rating: number;
    feedback: string;
    createdAt?: string;
}

export interface ReviewInput {
    senderId: number;
    recipientId: number;
    contractId: number;
    rating: number;
    feedback?: string;
}

File: FreelanceMarketplace/FreelanceMarketplace/GraphQL/Types/ReviewType.cs
using FreelanceMarketplace.Models;
using GraphQL.Types;

namespace FreelanceMarketplace.GraphQL.Types
{
    public class ReviewType : ObjectGraphType<Review>
    {
        public ReviewType()
        {
            Field(r => r.ReviewId);
            Field(r => r.Rating);
            Field(r => r.Feedback);

            Field<UserType>("users");
            //Field<ContractType>("contract").Resolve(context => context.Source.Contract);
        }
    }
}
